client/lib/domains/whois/protected-contact-information.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Internal dependencies
+ */
+import { isOpenHrsDomain, isOpenSrsDomain } from 'lib/domains';
+
+function getOpenHrsProtectedContactInformation( domain ) {
+	return {
+		firstName: 'Private',
+		lastName: 'Whois',
+		organization: 'Knock Knock Whois Not There LLC',
+		email: `${ domain }@privatewho.is`,
+		phone: '+1.8772738550',
+		address1: '60, 29th Street',
+		address2: '#343',
+		city: 'San Francisco',
+		state: 'CA',
+		postalCode: '94110-4929',
+		countryCode: 'US'
+	};
+}
+
+function getOpenSrsProtectedContactInformation( domain ) {
+	return {
+		firstName: '',
+		lastName: 'Contact Privacy Inc. Customer 012345',
+		organization: 'Contact Privacy Inc. Customer 012345',
+		email: `${ domain }@contactprivacy.com`,
+		phone: '+1.4165385457',
+		address1: '96',
+		address2: 'Mowat Ave',
+		city: 'Toronto',
+		state: 'ON',
+		postalCode: 'M6K 3M1',
+		countryCode: 'CA'
+	};
+}
+
+function getWwdProtectedContactInformation( domain ) {
```

Comment:
I do see a benefit tho of having the same approach to dealing with privacy with all APIs, instead of having some use the whois, some not. I'm fine with both.